HatekonyCPP/Fraction.h

```diff
@@ -0,0 +1,46 @@
+#pragma once
+
+#include <string>
+
+class Fraction
+{
+public:
+	static Fraction Parse(const std::string& value);
+
+public:
+	Fraction(int denominator);
+	Fraction(int numerator, int denominator);
+	Fraction(const double& decimalFraction);
```

Comment:
```suggestion
	Fraction(const double decimalFraction);
```
Elemi típusokat nem éri meg referenciaként átadni.